### ISSUE TITLE
Remove compareMCMCs

### ISSUE THREAD

**Issue by perrydv** [nimble-dev/nimble#1436]
I think we could remove the placeholder for the original `compareMCMCs` (just giving a message) that is now deprecated in favor of package `compareMCMCs` (which will get an update soon).

**Comment by danielturek** [comment]
@perrydv Agreed.

**Comment by paciorek** [comment]
Yes. I'll just put it on my to-do list and take care of with other minor things.

**Comment by paciorek** [comment]
Working on this now. 

@danielturek it appears on initial glance that the only use case for the `returnComponents` argument to `readBUGSmodel` was the now defunct `MCMCsuite`. What do you think about me removing that argument?

**Comment by danielturek** [comment]
@paciorek I, personally, think this `returnComponents` option is useful to have.  But that said, if the momentum is to clean up and remove it, I would understand.

**Comment by paciorek** [comment]
I'm happy to leave it.

Taking out the placeholders is done on devel, so closing this.
